### PR TITLE
T5 - Prime Number Checker

### PR DESCRIPTION
**Pull request by vikrampg1710-tmm**

### CODE REVIEW

File: Assignments/Program.cs [hunk 1]
```diff
@@ -1 +1,47 @@
...
+      string output;
+      int[] test = new int[] { 2, 3, 5, 11, 51, 61, 297, 313, 1381, 1663, 5157, 5153, 7871, 7873, 7919, 110503, 1257787, 3021379, 37156667 };
+      foreach (int i in test) {
+         Console.Write ($"{count++}. {i} ");
+         (output, Console.ForegroundColor) = IsPrime (i) ? ("Prime", Green) : ("Not a Prime", Red);
+         Console.WriteLine ($" - {output}");
+         Console.ResetColor ();
+      }
+      Console.Write ("\nNow, lets enter a number to check: ");
+      int s = Convert.ToInt32 (Console.ReadLine ());
```
Comment (asfar-nazar) [inline Assignments/Program.cs]: Use tryParse to validate if input is an integer

File: Assignments/Program.cs [hunk 2]
```diff
@@ -1 +1,47 @@
...
+      (output, Console.ForegroundColor) = IsPrime (s) ? ("Prime", Green) : ("Not a Prime", Red);
+      Console.WriteLine ($"{output}");
+      Console.ResetColor ();
+   }
+
+   /// <summary>Returns true if the input number is a prime, else false</summary>
+   public static bool IsPrime (long num) {
```
Comment (asfar-nazar) [inline Assignments/Program.cs]: IsPrime has as input arg 'long' but it is always called with argument 'int'

**Review by asfar-nazar — CHANGES_REQUESTED** [review]

File: Assignments/Program.cs (hunk 2 above)
Comment (vikrampg1710-tmm) [reply Assignments/Program.cs]: I presumed the user may input a large value, so tried to pass a long instead of int.

**Comment by vikrampg1710-tmm** [comment]
@asfar-nazar Your above suggestions have been implemented in the next commit 29454e6.  Please check.

**Review by asfar-nazar — APPROVED** [review]

**Pull request closed without merging**